debian/rules

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/make -f
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# The following packages explicitly state the version of the linux
```

Comment:
Is this comment correct? It seems misleading, or maybe I'm confused.

Reply:
Oh yeah, I've extracted it from the `delphix-package` project but forgot to update the comment. I'll do that.